### ISSUE TITLE
Theme toggle button is not accessible on screenreaders

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#4192]
**Describe the bug**
Our theme toggle button is not accessible to screen readers. The button can be tabbed to but gives no indication of what the button actually does.

**To Reproduce**

1. Go to [ethereum.org](https://ethereum.org)
2. Activate screen reader
3. Tab to theme toggle button

**Expected behavior**
Users should hear a audial indicator that this button toggles the website theme

**Screenshots**

<img width="444" alt="Screenshot 2021-10-11 at 11 20 08" src="https://user-images.githubusercontent.com/62268199/136775888-98efbe31-e547-4819-9c3b-b92c6d148569.png">
